src/components/common/button.tsx

```
@@ -0,0 +1,25 @@
+type ButtonProps = {
+    text: string;
+    onClick?: () => void;
+    disabled?: boolean;
+};
+
+const Button = ({ text, onClick, disabled = false }: ButtonProps) => {
+    return (
+        <div className="w-full max-w-[448px] absolute bottom-14 px-6">
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)